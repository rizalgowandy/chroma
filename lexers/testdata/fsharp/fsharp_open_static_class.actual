open type System.Math

module A =
    open type System.Math
