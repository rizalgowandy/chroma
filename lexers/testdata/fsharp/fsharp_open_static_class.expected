[
  {"type":"Keyword","value":"open type"},
  {"type":"Text","value":" "},
  {"type":"NameNamespace","value":"System.Math"},
  {"type":"Text","value":"\n\n"},
  {"type":"Keyword","value":"module"},
  {"type":"Text","value":" "},
  {"type":"NameNamespace","value":"A"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":"\n    "},
  {"type":"Keyword","value":"open type"},
  {"type":"Text","value":" "},
  {"type":"NameNamespace","value":"System.Math"},
  {"type":"Text","value":"\n"}
]
